next.config.js

```diff
@@ -29,15 +29,6 @@ module.exports = (phase, { defaultConfig }) => {
         test: /\.svg$/,
         use: "@svgr/webpack",
       })
-      config.module.rules.push({
-        test: /\.mp4$/,
-        use: {
-          loader: 'file-loader',
```

Comment:
I think we can remove `file-loader` from the deps

Reply:
Ah yes, good call, thanks!